cypress/integration/registration.spec.js

```diff
@@ -327,4 +327,49 @@ context('Actions', () => {
     cy.get('[data-testid=resource-tags]').should('contain', testTag2);
     cy.get('[data-testid=resource-tags]').should('contain', testTag3);
   });
+
+  it('register additional files', () => {
+    const testLink = 'http://www.test.com';
+    cy.get('[data-testid=new-registration-link]').click();
+    cy.get('[data-testid=new-resource-link]').click();
+    cy.get('[data-testid=new-resource-link-input]').type(testLink);
+    cy.get('[data-testid=new-resource-link-submit-button]').click();
+
+    cy.get('[data-testid=step-navigation-2]').click();
+    cy.route({
+      method: 'PUT',
+      url: 'https://file-upload.com/files/', // Must match URL set in mock-interceptor, which cannot be imported into a test
+      response: '',
+      headers: { ETag: 'etag' },
+    });
+    cy.get('[data-testid=additional-files-uppy-dashboard] input[type=file]:first-of-type').uploadFile(
+      'testPicture.png'
+    );
+    cy.get(`[data-testid=additional-file-content-${mockContent.identifier}]`).contains(
+      mockContent.features.dlr_content
+    );
+    cy.get(`[data-testid=thumbnail-${mockContent.identifier}]`).should('exist');
+    cy.get(`[data-testid=additional-file-${mockContent.identifier}-delete-button]`).click();
+    cy.get(`[data-testid=thumbnail-${mockContent.identifier}]`).should('not.exist');
+  });
+
+  it('register thumbnail', () => {
+    cy.get('[data-testid=new-registration-link]').click();
```

Comment:
Vet ikke om det er viktig å teste men her er noen krav til denne funksjonaliteten:
Fila som kan lastes opp skal ha mime type image, ellers så skal man ikke få lov til å laste den opp.
Hvis man har lastet opp bilde før, så skal man få valg om å fjerne forrige bilde eller laste opp nytt.
Mens bildet lastes opp skal knappen for å laste opp nytt bilde være disablet.

Reply:
Mime-type-sjekk lagt inn. Sletting blir sjekket.  Slik mockingen er lagt opp, får jeg ikke testet opplasting når man _ikke_ har bilde fra før
Sjekk av disabled knapp tror jeg blir mye jobb for noe som ikke er alt for viktig.